ReleaseHistory.md

```diff
@@ -16,6 +16,7 @@
 - NEW => new feature 
 
 ## UNRELEASED
+* NEW: `BA2004.EnableSecureSourceCodeHashing` Provide actual hash in BA2004 reporting of native binaries instead of just saying insecure hash. [926](https://github.com/microsoft/binskim/pull/926)
```

Comment:
>NEW [](http://example.com/codeflow?start=2&length=3)

This release note isn't a complete sentence.

`BA2004.EnableSecureSourceCodeHashing` now explicitly reports the insecure hash algorithm found (or reports no hash information was present in that circumstance). [926](https://github.com/microsoft/binskim/pull/926)
* NR :